readme.md

```diff
@@ -1,7 +1,7 @@
 # Frontend practice with catalog page
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_catalog/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_catalog/report/html_report/)
+- [DEMO LINK](https://yulabakun.github.io/layout_catalog/)
```

Comment:
Link doesn't work

Reply:
There was some sort of a problem on GitHub side. I redeployed, and now it works.
![image](https://github.com/mate-academy/layout_catalog/assets/88306382/db859b7f-4939-458c-8705-77fb14e68baf)